op-e2e/actions/upgrades/isthmus_fork_test.go

```diff
@@ -364,14 +364,14 @@ func TestIsthmusNetworkUpgradeTransactions(gt *testing.T) {
 	require.NotEmpty(t, txn.Data(), "upgrade tx must provide input data")
 
 	// EIP-2935 contract is deployed
-	expectedBlockHashAddress := crypto.CreateAddress(derive.BlockHashDeployerAddress, 0)
+	expectedBlockHashAddress := crypto.CreateAddress(predeploys.EIP2935ContractDeployer, 0)
 	require.Equal(t, predeploys.EIP2935ContractAddr, expectedBlockHashAddress)
 	code := verifyCodeHashMatches(t, ethCl, predeploys.EIP2935ContractAddr, predeploys.EIP2935ContractCodeHash)
 	require.Equal(t, predeploys.EIP2935ContractCode, code)
 
 	// Test that the beacon-block-root has been set
 	checkRecentBlockHash := func(blockNumber uint64, expectedHash common.Hash, msg string) {
-		historyBufferLength := uint64(8191)
+		historyBufferLength := uint64(params.HistoryServeWindow) - 1
```

Comment:
Let's keep it like the constant. The constant doc only refers to the block-hash history EIP, not the beacon-block-root EIP. I don't like that they're coupling those like that.